SampleProject/SampleProject.xcodeproj/project.pbxproj

```diff
@@ -727,6 +731,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = "";
```

Comment:
I'm not sure how to prevent this from being committed along with the other changes in this file, or if it even matters...